polaris-react/UNRELEASED.md

```diff
@@ -24,6 +24,7 @@ Use [the changelog guidelines](/documentation/Versioning%20and%20changelog.md) t
 
 - Removed `lodash` ([#5544](https://github.com/Shopify/polaris/pull/5544))
 - Uses more permissive dependency for `@types/react` and `@types/react-dom` ([#5575](https://github.com/Shopify/polaris/pull/5575))
+- Integrated the `@shopify/polaris-tokens` CSS stylesheet in the `CustomProperties` component ([#5659](https://github.com/Shopify/polaris/pull/5659))
```

Comment:
Apollogies, I didn't look where this changelog entry this was under in my original suggestion.

Since there is quite a few changes in the PR I suggest we move this entry up to the `Enhancements` heading and adding a "Upgraded `@shopify/polaris-tokens` to `v5.0.0`" entry for this heading `Dependency upgrades`.

Reply:
Perhaps we should also include some deprecation notices for removing the `JSON` token-groups and the `src/tokens/tokens.ts` exports. They weren't documented as public APIs but definitely accessible:
- https://unpkg.com/browse/@shopify/polaris@9.0.0/build/esm/tokens/token-groups/
- https://unpkg.com/browse/@shopify/polaris@9.0.0/build/esm/tokens/tokens.js